tests/test.rng

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
this seems to be a basic schema definition that is included in more specific schemas like `tests/core/attributes/lang/schema.rng` for testing use cases; therefore rename to `base.rng` or `test-base.rng`?